python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -5688,3 +5689,174 @@ def test_dataframe_assign_scalar_with_scalar_cols(col_data, assign_val):
     gdf["b"] = assign_val
 
     assert_eq(pdf, gdf)
+
+
+# flake8: noqa : W291
```

Comment:
Does the actual print formatting require trailing whitespace?

Reply:
We'll need to add spaces, but trailing spaces are not needed. Hence now doing a `rstrip`